packages/store/gas-report.json

```diff
@@ -303,7 +303,7 @@
     "file": "test/GasStorageLoad.t.sol",
     "test": "testCompareStorageLoadMUD",
     "name": "MUD storage load field (warm, 1 word)",
-    "gasUsed": 245
+    "gasUsed": 2300
```

Comment:
do we know where this is coming from? 🤔

Reply:
given that this increase isn't showing up in ~every other gas report, I am guessing something unrelated is going on